cmd/odf/restore/crds.go

```diff
@@ -1,12 +1,54 @@
 package restore
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
 	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
 	pkgrestore "github.com/rook/kubectl-rook-ceph/pkg/restore"
 	"github.com/spf13/cobra"
 )
 
+// groupVersions defines the supported CRD groups and their corresponding API versions.
+var groupVersions = map[string]string{
+	"ocs.openshift.io":       "v1",
+	"ceph.rook.io":           "v1",
+	"storage.k8s.io":         "v1",
+	"csi.ceph.io":            "v1",
```

Comment:
In 4.19 we are pushing for v1beta1 for the csi CRDs, let's assume that for now
```suggestion
	"csi.ceph.io":            "v1beta1",
```

Reply:
done